packages/clerk-js/bundlewatch.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "files": [
-    { "path": "./dist/clerk.browser.js", "maxSize": "65kB" },
+    { "path": "./dist/clerk.browser.js", "maxSize": "64.5kB" },
```

Comment:
I'll be working on UI components, and i want to closely monitor this change. Lowering the limit will help with that.